src/modal/modal.scss

```diff
@@ -28,6 +28,7 @@
     z-index: 1000;
     max-width: 50%;
     min-width: 400px;
+    max-height: 100%;
```

Comment:
Just curious, how do you test that this works?

Reply:
I first encountered the bug when viewing the modal demo on my phone in landscape.  To test, I inspected the page & then:

1. Press the button in devtools
2. Select a device with a small screen size, I picked iPhone 5.
3. Press the rotate button.

![Screen Shot 2021-12-22 at 4 48 14 PM](https://user-images.githubusercontent.com/849817/147158951-bba4adf8-0436-4386-85b2-4e5b3daa3817.png)
 
You may need to enable **Show device frame** from this menu:

![Screen Shot 2021-12-22 at 4 49 36 PM](https://user-images.githubusercontent.com/849817/147159087-364d1e0d-bb93-425e-9595-6e9af4fe4945.png)